kotlin/src/test/kotlin/net/jqwik/kotlin/CombinatorDslTests.kt

```diff
@@ -106,4 +107,17 @@ class CombinatorDslTests {
 
         assertThat(combined.sample()).isEqualTo(3)
     }
+
+    @Property
```

Comment:
I tend to reduce the number of tries for these kinds of shrinking tests to 10 or 100 max, which makes certain that not only edge cases have been created and shrinking works in general.

Reply:
Oh oops, I overlooked the `@PropertyDefaults` in the class I blatantly copied from, my bad